src/browser/main.coffee

```diff
@@ -57,11 +57,14 @@ handleStartupEventWithSquirrel = ->
 setupCrashReporter = ->
   crashReporter.start(productName: 'Atom', companyName: 'GitHub')
 
-setupAtomHome = ->
+setupAtomHome = (args) ->
```

Comment:
It looks like args is no longer used here so would you mind backing out this change here?